src/components/routes/Search.tsx

```diff
@@ -83,4 +86,12 @@ class Search extends PureComponent<FullProps> {
     }
 }
 
-export default MetadataResolver<FullProps>(QUERY_COLLECTION_PROPERTIES, QUERY_COLLECTION_VALUES)(Search);
\ No newline at end of file
+const dataResolver = compose<ComponentType<{}>>(
+    withRouter,
+    metaDataResolver(QUERY_COLLECTION_PROPERTIES),
+    renderLoader('metadata'),
```

Comment:
If you make the loader accept either a string or an array of strings, you only need to add the loader once as a hoc, instead of multiple times

Reply:
Thinking about it a bit more, I realised renderLoader does more than just... render a loader. It actually blocks the next HoC from rendering.

Say for example I'd change this line with `renderLoader(['metadata', 'data'])`: the loader would keep showing forever, because the data query never actually starts. (because the component never mounts).
Yes, I could move it to the last argument of the compose call instead, but then `QUERY_COLLECTION_VALUES` would crash because it expects the metadata prop to be passed to it.